docs/command-line-interface.md

```diff
@@ -798,7 +798,7 @@ polaris namespaces list --catalog my_catalog --parent a.b
 
 ### Privileges
 
-The `privileges` command is used to grant various privileges to a catalog role, or to revoke those privileges. Privileges can be on the level of a catalog, a namespace, a table, or a view. For more information on privileges, please refer to the [docs](./entities.md#privilege).
+The `privileges` command is used to grant various privileges to a catalog role, or to revoke those privileges. Privileges can be on the level of a catalog, a namespace, a table, or a view. For more information on privileges, please refer to the [docs](https://polaris.io/#tag/Polaris-Catalog-Entities/Privilege).
```

Comment:
I think it would be better to wait the documentation moved to polaris.apache.org (else we will have to change again).
Even if `polaris.io` will redirect to `polaris.apache.org`, I think it's cleaner to directly link to `polaris.apache.org`.